### DISCUSSION TITLE
Impact of Declining Vaccine Efficacy With Time

### DISCUSSION THREAD

**Discussion by nh905** [microCOVID/microCOVID#1105] [Ideas]
Has there been any discussion on factoring in the time since the individual was fully vaccinated?  There have been a lot of contradictory reports about reduced efficacy around the six month point and breakout infections among vaccinated people, although I suspect some of the issues are with the analysis.

**Comment by blanchardjeremy** [comment]
Yep! We have an issue open for it here: https://github.com/microCOVID/microCOVID/issues/999

The [Isreal study](https://www.mattbell.us/delta-and-long-covid/) is the place where we get the "4-6 month" data point.

I don't think anyone on our team has done any more analysis on it yet

:)

**Reply by nh905** [reply]
I had a brief look at the Israel study which seemed quite pessimistic.  I also came across [https://www.covid-datascience.com/post/israeli-data-how-can-efficacy-vs-severe-disease-be-strong-when-60-of-hospitalized-are-vaccinated](https://www.covid-datascience.com/post/israeli-data-how-can-efficacy-vs-severe-disease-be-strong-when-60-of-hospitalized-are-vaccinated) which reanalyzed the data and came to different conclusions.  At this point, I defer to smarter people to figure out who is closer to the truth.  One of the strengths of microcovid.org is its ability to cut through the chatter and deliver actionable information.
